extras/module-node/duk_module_node.c

```diff
@@ -0,0 +1,248 @@
+/*
+ * Node.js-like require() and module loader
+ *
+ * https://nodejs.org/api/modules.html
+ */
+
```

Comment:
Trivia: there are two basic comment styles in the Duktape code base. First style is "compact":

    /* Foo bar.
     * Quux.
     */

In this style there's a single space between the star and the text. The second style is a "banner":

    /*
     *  Two spaces.
     */

Here there are two spaces before the text. This is pretty arbitrary but that's the current convention in any case. The basic intent is to visualize separate the banner as a block separate, and the compact comment as applying to the next function, variable, etc (and there's usually no empty line between the compact comment and the function/variable).